packages/rocket-uploads3-store-event-aws-infrastructure/src/lambdas/index.ts

```diff
@@ -0,0 +1,44 @@
+import { AWSError, DynamoDB } from 'aws-sdk'
+import { PutItemOutput } from 'aws-sdk/clients/dynamodb'
+
+export const handler = async (event: any): Promise<any> => {
```

Comment:
Just missing this type above `(event: any)`

Reply:
Which makes me think, perhaps it should be `unknown`, rather than `any`?

Reply:
There are types for every AWS event, there must be something like `S3FileUploaded`, for example. 

Regarding the response, I think there must be one. If not, we can use there `unknown` as nick suggeted

Reply:
@juanjoman @alvaroloes @NickSeagull 

- Believe it or not I could not find any reference what kind of type should I use for the S3 event to Lambda event 😢 

- All the references codes when receiving an S3 notification are not returning any value which I think makes sense. We don't control the response since it's automatically triggered from S3. We can't do anything with that response. 

- I can't return a Promise<unknown> the error is get is `A function whose declared type is neither 'void' nor 'any' must return a value.` Sorry still a noob in TS not sure what to do here.

Reply:
Hmm I'm not sure about the input (event param) but for the return type, I think it would be `Promise<void>` if you don't return that await, and `Promise<DynamoDB.PutItemOutput>` 🤔

Reply:
Yep! What Juanjo said @rdoria1